subsystem/src/manager.rs

```diff
@@ -29,13 +29,13 @@ pub struct ManagerConfig {
     /// Subsystem manager name
     name: &'static str,
     /// Shutdown timeout. Set to `None` for no (i.e. unlimited) timeout.
-    shutdown_timeout: Option<Duration>,
+    shutdown_timeout_per_subsystem: Option<Duration>,
 }
 
 impl ManagerConfig {
     /// Default shutdown timeout.
     const DEFAULT_SHUTDOWN_TIMEOUT: Option<Duration> = if cfg!(all(feature = "time", not(loom))) {
-        Some(Duration::from_secs(20))
+        Some(Duration::from_secs(5))
```

Comment:
I would make this 30 seconds or even more. Otherwise if there's an issue we won't notice it. This should never happen!